scapy/contrib/dce_rpc.py

```diff
@@ -45,8 +42,13 @@ def set_endianess(self, pkt):
         """Add the endianness to the format"""
         end = self.endianess_from(pkt)
         if isinstance(end, str) and len(end) > 0:
```

Comment:
Could you use the opportunity to change that to:
```Suggestion
         if isinstance(end, str) and end:
```

Reply:
Added.